.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: Report an issue about using Custodian
+labels: [kind/bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Tell us what happened!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: expected-bahavior
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what you expected to happen.
+      placeholder: What were you expecting to happen?
+      value: "It did this instead of this"
+    validations:
+      required: true
+  - type: textarea
+    id: version
+    attributes:
+      label: Cloud Custodian version and dependency information
+      description: Please run `custodian version --debug` and paste the output here
+      render: shell
+  - type: dropdown
+    id: cloud-provider
+    attributes:
+      label: Cloud Provider
+      description: What cloud provider are you using?
```

Comment:
If you want to be neurotic, you can use "which" instead of "what." ("Which" is the more formal word to use when asking a user to pick from a specific list of options)

Does this allow for multiple selections? Is that even necessary?